src/components/Drawer/Drawer.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import bem from 'bem';
+import { IoIosArrowUp } from 'react-icons/io';
+import styles from './Drawer.scss';
+
+const { block, elem } = bem({
+    name: 'Drawer',
+    classnames: styles,
+    propsToMods: ['isOpen'],
+});
+
+const Drawer = props => {
+    const { title, initialIsOpen, children, ...rest } = props;
+    const [isOpen, setIsOpen] = useState(initialIsOpen);
+
+    return (
+        <div {...block({ isOpen })} {...rest}>
+            <header {...elem('head', props)}>
+                <h4 {...elem('head-title', props)}>{title}</h4>
```

Comment:
we're using camelCase style for all of the styling purposes, right?

Reply:
Probably yes. I was confused while I was implementing Drawer component because I looked at some last components (LocationCard), and some of them also are mixed style. So I thought for _block_  we use camelCase, for _element_ hyphen, for _modificator_  camelCase. I agree with you, we should stick one style
![Screenshot 2019-09-10 at 10 19 52](https://user-images.githubusercontent.com/19436304/64598282-c17a3b80-d3b7-11e9-8228-d723aa0fdd24.png)